build/gen_stub.php

```diff
@@ -399,6 +510,10 @@ public function getArgInfoName(): string {
         return "arginfo_$underscoreName";
     }
 
+    public function getMethodsynopsisFilename(): string {
+        return implode('_', $this->name->parts);
+    }
```

Comment:
This method is out of place here imho.

Reply:
Yes, I was also not very happy with it, however the interface doesn't let me to access `$this->name->parts` directly. And since the implementation is different in case of the `MethodName` class, I think we'll need a method similar to `getMethodsynopsisFilename` in the `FunctionOrMethodName` interface. Or do you have other suggestion?